app/src/main/java/de/westnordost/streetcomplete/quests/hairdresser/Hairdresser.kt

```diff
@@ -0,0 +1,8 @@
+package de.westnordost.streetcomplete.quests.hairdresser
+
+enum class Hairdresser(val isMale: Boolean, val isFemale: Boolean) {
+    NOT_SIGNED(false, false),
+    ONLY_FEMALE(false, true),
+    ONLY_MALE(true, false),
+    MALE_AND_FEMALE(true, true),
+}
```

Comment:
Why isMale=false, isFemaile=false for "not signed"?

You should use a sealed interface here, i.e. 
sealed interface HairdresserAnswer
enum class Hairdresser : HairdresserAnswer { ... }
object HairdresserNotSigned : HairdresserAnswer

Reply:
Hm, because those flags are about what **is** signed? 

So if only "for male" is signed on the hairdresser, only `isMale=true` (as it was signed), and `isFemale=false` (as it was **not** signed). Same logic applies if nothing is signed: both `isMale=false` and `isFemale=false`, because none of them were signed.

But if you insist, I'll investigate how to use that `sealed interface` thingy and use that (only for NOT_SIGNED or for all of them?)

Reply:
Well, if you think it is fine this way, we can leave it at that. However, since you haven't used sealed interfaces so far, maybe you should look at some other quests that use it and only decide then whether that approach would not be cleaner.